packages/p2p/src/components/table/infinite-loader-list.jsx

```diff
@@ -40,7 +40,13 @@ export const InfiniteLoaderList = ({
     return (
         <InfiniteLoader isItemLoaded={index => index < items.length} itemCount={item_count} loadMoreItems={loadMore}>
             {({ onItemsRendered, ref }) => (
-                <AutoSizer style={{ height: initial_height || 600 }}>
+                <AutoSizer
+                    style={{
+                        // screen size - header size - footer size - page overlay header - page overlay content padding -
+                        // tabs height - table header height
+                        height: initial_height || 'calc(100vh - 48px - 36px - 41px - 2.4rem - 36px - 52px)',
```

Comment:
these used in for so many times, probably we can make a constant for it?

Reply:
@mustofa-binary Same as above.